docroot/themes/contrib/civictheme/theme-settings.php

```diff
@@ -634,3 +635,388 @@ function _civictheme_path_field_description($original_path, $fallback_path) {
     '@local-file' => $local_file,
   ]);
 }
+
+/**
+ * Provide color settings to theme settings form.
+ *
+ * @SuppressWarnings(PHPMD.UnusedFormalParameter)
+ * @SuppressWarnings(PHPMD.ExcessiveMethodLength)
+ */
+function _civictheme_form_system_theme_settings_colors(&$form, FormStateInterface &$form_state) {
+  $form['colors'] = [
+    '#type' => 'details',
+    '#title' => t('Colors'),
+    '#weight' => 40,
+    '#open' => TRUE,
+    '#tree' => TRUE,
+  ];
+
+  $form['colors']['use_brand_colors'] = [
+    '#title' => t('Use Brand colors'),
+    '#type' => 'checkbox',
+    '#default_value' => theme_get_setting("colors.use_brand_colors") ?? TRUE,
+  ];
+  // Brand.
+  $form['colors']['brand'] = [
+    '#type' => 'fieldset',
+    '#title' => t('Brand colors'),
+    '#tree' => TRUE,
+    '#states' => [
+      'visible' => [
+        ':input[name="colors[use_brand_colors"]' => ['checked' => TRUE],
+      ],
+    ],
+  ];
+
+  // Brand light.
+  $form['colors']['brand']['light'] = [
+    '#type' => 'fieldset',
+    '#tree' => TRUE,
+    '#attributes' => [
+      'class' => 'color-light',
+    ],
+  ];
+  $form['colors']['brand']['light']['brand1'] = [
```

Comment:
@joshua-salsadigital 
please refactor to use a loop for consistency

Reply:
Fixed